src/main/java/gov/usgs/ngwmn/logic/WaterLevelMonthlyStats.java

```diff
@@ -0,0 +1,53 @@
+package gov.usgs.ngwmn.logic;
+
+import static gov.usgs.wma.statistics.app.Properties.*;
+
+import java.util.List;
+import java.util.function.Function;
+
+import gov.usgs.ngwmn.model.MediationType;
+import gov.usgs.ngwmn.model.WLSample;
+import gov.usgs.wma.statistics.app.Properties;
+import gov.usgs.wma.statistics.logic.MonthlyStatistics;
+import gov.usgs.wma.statistics.logic.StatisticsCalculator;
+import gov.usgs.wma.statistics.model.JsonDataBuilder;
+
+public class WaterLevelMonthlyStats extends MonthlyStatistics<WLSample> {
```

Comment:
This was formerly an inner class that I felt should be promoted to a top level class at this time,